difflint/lib.py

```diff
@@ -13,6 +14,29 @@
 MISSING_FILE_EXIT_CODE = 72  # os.EX_OSFILE is not portable
 
 
+def make_path_from_cwd(*args):
+    """
```

Comment:
Make difflint `pep8` compliant!? What madness is this!? (I will make this change; thank you.)